content/200-concepts/100-components/250-preview-features/080-cli-preview-features.mdx

```diff
@@ -6,14 +6,16 @@ tocDepth: 2
 
 <TopBlock>
 
-<Admonition type="info">
+When we release a new Prisma Client or schema feature, it often starts in preview so that you can test it and submit your feedback. After we  improve the feature with your feedback and are satisfied with the internal test results, we promote the feature to generally availability.
```

Comment:
```suggestion
When we release a new Prisma CLI feature, it often starts in preview so that you can test it and submit your feedback. After we improve the feature with your feedback and are satisfied with the internal test results, we promote the feature to general availability.
```